doc/manuals/contribution_guidelines.md

```diff
@@ -19,6 +19,27 @@ planning to contribute to the code you should read this document and get familia
 Note that contribution workflows themselves (e.g. pull requests, etc.) are described in another document 
 ([FIWARE Development Guidelines](https://forge.fiware.org/plugins/mediawiki/wiki/fiware/index.php/Developer_Guidelines)).
 
+## Pull Request protocol
+
+As explained in ([FIWARE Development Guidelines](https://forge.fiware.org/plugins/mediawiki/wiki/fiware/index.php/Developer_Guidelines)) 
+contributions are done using a pull request (PR). The detailed "protocol" used in such PR is described below:
+
+* Direct commits to master branch (even single-line modifications) are not allowed. Every modification has to come as a PR
+* In case the PR is implementing/fixing a numbered issue, the issue number has to be referenced in the body of the PR at creation time
+* Anybody is welcome to provide comments in the PR (either direct comments or using the review feature offered by Github)
```

Comment:
to the PR

Reply:
Fixed in 83df10cc1acd354fce084e7f7e40f3d5f1fd14ad